webpack.config.js

```diff
@@ -38,7 +40,11 @@ const config = {
       new BundleAnalyzerPlugin({
         analyzerMode: 'disabled',
         generateStatsFile: true
-      })
+      }),
+      new webpack.DefinePlugin({
+        APP_VERSION: JSON.stringify(JSON.parse(fs.readFileSync('./package.json')).version),
+        CIRCLE_SHA1: JSON.stringify(process.env.CIRCLE_SHA1 || '-')
```

Comment:
CIRCLE_SHA1 is unlikely to work here. Unlike the kaios app, this package is not produced in CI but locally. 

You can use `git rev-parse --short HEAD` to get the sha1.

Reply:
Awesome, thanks! Updated now